### PR TITLE
Update realtime_servo_tutorial.rst

### PR DESCRIPTION
**Pull request by devvaibhav455**
### Description

Although, the code suggests cloning the old version of UR's repo but it does not clearly mention it in words. I did not clone the old version because I already had the latest version and the tutorial didn't work for me for now obvious reasons. So, I am adding a note in bold so that no one else does the same mistake as mine and don't waste 2 days like me. I also added a debug tip to see when one should expect the tutorial to work.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: doc/realtime_servo/realtime_servo_tutorial.rst [hunk 1]
```diff
@@ -17,7 +17,10 @@ Getting Started
 ---------------
 This tutorial demonstrates the servo node with a UR5 Gazebo simulation. If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
 
-Clone `universal_robot melodic-devel branch <https://github.com/ros-industrial/universal_robot.git>`_ into the same catkin workspace from `Getting Started`: ::
+Clone `universal_robot melodic-devel branch <https://github.com/ros-industrial/universal_robot.git>`_ into the same catkin workspace from `Getting Started`:
+
+**NOTE: Please stick to older version of Universal Robot Github (1.2.7 ) as newer versions (1.3.x) are incompatible with the servo tutorial.** (`Src <https://github.com/ros-planning/moveit_tutorials/commit/f9fba77cd6cab31c9ae360bbaca3896a6acef446>`_)
```
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:22]: I'd recommend deleting this link. I don't think it adds anything, really, and certainly needs a better description than "Src". Looks good to me besides that. Thanks!

```suggestion
**NOTE: Please stick to older version of Universal Robot Github (1.2.7 ) as newer versions (1.3.x) are incompatible with the servo tutorial.**
```

**Comment by devvaibhav455** [comment]
Hello Dr. @AndyZe  : I made the suggested changes. Were you able to check them? Happy July 4 btw.

**Review by AndyZe — APPROVED** [review]

File: doc/realtime_servo/realtime_servo_tutorial.rst [hunk 2]
```diff
@@ -126,3 +129,8 @@ There is a Python integration test in ``test/integration``. Run it by:
 
   roscd moveit_servo
   catkin run_tests --this
+  
+Debug Tips
+-------------------
+Do run either ``rosservice call /controller_manager/list_controllers`` or ``rosrun controller_manager controller_manager list``. For moveit_servo to be working, ``joint_group_position_controller``(tested) or ``joint_group_velocity_controller``(I think, haven't tested personally this one though) should be ``running``
+
```
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:136]:
```suggestion
```

File: doc/realtime_servo/realtime_servo_tutorial.rst [hunk 3]
```diff
@@ -126,3 +129,7 @@ There is a Python integration test in ``test/integration``. Run it by:
 
   roscd moveit_servo
   catkin run_tests --this
+
```
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:132]:
```suggestion
 
```

File: doc/realtime_servo/realtime_servo_tutorial.rst [hunk 4]
```diff
@@ -126,3 +129,7 @@ There is a Python integration test in ``test/integration``. Run it by:
...
```
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:132]:
```suggestion

```

File: doc/realtime_servo/realtime_servo_tutorial.rst [hunk 5]
```diff
@@ -126,3 +129,6 @@ There is a Python integration test in ``test/integration``. Run it by:
 
   roscd moveit_servo
   catkin run_tests --this
+Debug Tips
+-------------------
+Do run either ``rosservice call /controller_manager/list_controllers`` or ``rosrun controller_manager controller_manager list``. For moveit_servo to be working, ``joint_group_position_controller``(tested) or ``joint_group_velocity_controller``(I think, haven't tested personally this one though) should be ``running``
```
Comment (rhaschke) [inline doc/realtime_servo/realtime_servo_tutorial.rst:134]: This should be validated and correspondingly rephrased before merging.

**Review by rhaschke — COMMENTED** [review]
Please verify that `joint_group_velocity_controller` works as well and rephrase the debug comment.

File: doc/realtime_servo/realtime_servo_tutorial.rst (hunk 5 above)
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:134]:
```suggestion
Do run either ``rosservice call /controller_manager/list_controllers`` or ``rosrun controller_manager controller_manager list``. For moveit_servo to be working, ``joint_group_position_controller`` or ``joint_group_velocity_controller`` should be ``running``.
```

File: doc/realtime_servo/realtime_servo_tutorial.rst [hunk 6]
```diff
@@ -126,3 +129,6 @@ There is a Python integration test in ``test/integration``. Run it by:
 
   roscd moveit_servo
   catkin run_tests --this
+Debug Tips
+-------------------
```
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:133]:
```suggestion
  catkin run_tests --this

Debug Tips
---------------
```

**Review by rhaschke — APPROVED** [review]
Approving if .rst formatting issue reported by CI is fixed.

**Pull request merged**